core-toolbar.css

```diff
@@ -87,20 +111,24 @@ polyfill-next-selector { content: ':host.core-narrow .toolbar-tools, .core-narro
   left: 0;
 }
 
+/* ******************************* */
+/* TODO: Should content be styled? */
```

Comment:
I wanted to remove it but wanted to run it by you and others first. I'm a fan of "no" as well since that's what I expect from using any random custom element: don't touch my light DOM.